### PR TITLE
[FIX]'Thanks for posting' message occurred even user can't post anything

### PR DESCRIPTION
**Pull request by tde-banana-odoo**
Task: https://www.odoo.com/web#id=11862&view_type=form&model=project.task&menu_id=3942&action=327
Pad: https://pad.odoo.com/p/r.MKnetvwi6XdN13Qm

### CODE REVIEW

File: addons/website_forum/static/src/js/website_forum.share.js [hunk 1]
```diff
@@ -37,10 +37,20 @@
     website.ready().done(function() {
 
         // Store social share data to display modal on next page
-        $(document.body).on('click', '.oe_social_share_call', function() {
-            var social_data = {};
-            social_data['target_type'] = $(this).data('social-target-type');
-            sessionStorage.setItem('social_share', JSON.stringify(social_data));
+        $('.oe_social_share_call').parent().submit(function(event) {
```
Comment (tde-banana-odoo) [inline addons/website_forum/static/src/js/website_forum.share.js]: Nope. There is a much easier way to do this: check that the element has not the class "karma_required". I will fix directly in saas-6.

**Comment by tde-banana-odoo** [comment]
Nope. There is a much easier way to do this: check that the element has not the class "karma_required". I will fix directly in saas-6.

File: addons/website_forum/static/src/js/website_forum.share.js (hunk 1 above)
Comment (apa-odoo) [inline addons/website_forum/static/src/js/website_forum.share.js]: @tde-banana-odoo yeah! it's work if there is class "karma_required", its good. but if there is error at python side and Traceback(mean controller not return success) then 'Thanks for posting' message will occurred when page is refreshed.

Comment (tde-banana-odoo) [inline addons/website_forum/static/src/js/website_forum.share.js]: Why should an error occur ?

Comment (apa-odoo) [inline addons/website_forum/static/src/js/website_forum.share.js]: @tde-banana-odoo  at current scenario there is not any error. but, suppose in some case there is error then message will occured.

**Pull request closed without merging**